Automattic/wp-calypso
# Override post editor sidebar nav close button when launchpad_screen option is full

#### Proposed Changes

* Update ETK post editor sidebar close url to redirect users to the launchpad screen when the `launchpad_screen` option = `full`

#### Testing Instructions



* Checkout this branch
* Sandbox public api and your new launchpad enabled site
* Navigate to the `apps/editing-toolkit` within wp-calypso and run: `yarn dev --sync`
* Navigate to `{siteSlug}/wp-admin/post.php` and edit an existing post or create a new one.
* Open the sidebar and confirm the exit/close button says `Next steps` and navigates to the launchpad screen
<img width="255" alt="image" src="https://user-images.githubusercontent.com/10482592/214925805-9941e417-2761-4190-8e8c-0b20162ed929.png">
* Follow the instructions to upload the ETK plugin to your atomic site: PCYsg-ly5-p2 and repeat the step above.



#### Pre-merge Checklist





Related to #72394
